src/xdocs/checks/javadoc/javadocpackage.xml

```diff
@@ -45,7 +43,7 @@
               <td>Specify the file type extension of files to process.</td>
               <td><a href="../../property_types.html#String.5B.5D">String[]</a></td>
               <td><code>.java</code></td>
-              <td>5.0</td>
+              <td>4.4</td>
```

Comment:
Link to commit already in PR description.
Pom version for this commit:
https://github.com/checkstyle/checkstyle/blob/d4e32973ad400ec7b4d2b908c6eb5ad3a621b960/pom.xml#L5

Reply:
JavadocPackageCheck did not come out until 5.0 .
https://github.com/checkstyle/checkstyle/blob/22c0361c6efafbce9972ebbe005a36090d1724e6/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocPackageCheck.java#L68

Reply:
In this case I don't know how to trust any approach. There was some mess, I recommend to not be super precise on this since, and in case of uncertainty, put 5.0

Reply:
reverted, map is used.